src/lib/FieldType/Mapper/AbstractRelationFormMapper.php

```diff
@@ -17,6 +17,8 @@
 
 abstract class AbstractRelationFormMapper implements FieldValueFormMapperInterface
 {
+    protected const SELECTION_SELF = -1;
```

Comment:
Does this differs from `RelationType::LOCATION_SELF` logic wise?

Reply:
Yes. The `-1`s are used to indicate the same behaviour but one is just radio button value and other is actual storage value for location.